collectors/conf/spark.conf

```diff
@@ -0,0 +1,31 @@
+[base]
+enabled: True
+interval: 120
+
```

Comment:
why 120 seconds, is each collecting too expensive? I feel 120 seconds is too long.